recipes/pattern-causality/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set name = "pattern-causality" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/skstavroglou/pattern_causality_py/releases/download/v{{ version }}/pattern_causality-{{ version }}.tar.gz
+  sha256: 99d42253d559992eb0137356ad52036789f00dd87ee7f5b1e4e6ebbad5e55141
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: >-
+    {% if target_platform == "osx-64" %}
+    export SDKROOT=$(xcrun --sdk macosx --show-sdk-path) && export CC=${CC} && export CXX=${CXX} && export CFLAGS="${CFLAGS} -isysroot ${SDKROOT} -I${SDKROOT}/usr/include -mmacosx-version-min=10.9 -Xpreprocessor -fopenmp" && export CXXFLAGS="${CXXFLAGS} -stdlib=libc++ -isysroot ${SDKROOT} -I${SDKROOT}/usr/include -I${SDKROOT}/usr/include/c++/v1 -mmacosx-version-min=10.9 -Xpreprocessor -fopenmp" && export LDFLAGS="${LDFLAGS} -stdlib=libc++ -isysroot ${SDKROOT} -mmacosx-version-min=10.9 -lomp" && {{ PYTHON }} -m pip install . --no-deps -vv
+    {% else %}
+    {{ PYTHON }} -m pip install . --no-deps -vv
+    {% endif %}
```

```suggestion
  script: {{ PYTHON }} -m pip install . --no-deps -vv
```
I do not know why you are setting all of these variables in your script on macOS.  `SDKROOT` should already be defined in our environment as should most of these other flags.  Please remove this and we can see how the builds runs without it.  If any changes are needed, I expect they should be much less than what is currently being done here.